Assets/CohortUnityClient/Scripts/CHSession.cs

```diff
@@ -115,8 +140,181 @@ public class CHSession : MonoBehaviour {
 
     private bool socketConnectionActive = false;
 
+    private string jwtToken = "";
+    private UnityWebRequest cohortUpdateEventRequest;
+
+    private List<CHEpisode> episodesArray;
+
+    private string episodeJson;
+    private Boolean successfulServerRequest;
+
+    [Serializable]
+    public class JwtToken
+    {
+      public string jwt;
+    }
+
+
+    [Serializable]
+    public class CHEpisode
+    {
+      public int episodeNumber { get; set; }
+      public string label { get; set; }
+      public List<CHMessage> cues { get; set; }
+    }
+
+
+
+    string serverToEventUrl(string url) {
+      //this checks for any of the following words/letter sequences, but I'm not sure how to use /(localhost|.local|192.168.)/mi in this context
+      string urlInput = "(localhost|.local|192.168.)"; 
```

Comment:
Ok, I'll have a look. The `/mi` is two flags or options, `/i` for case-insensitive and `/m` for multiline which we actually don't need. It looks like Microsoft implemented these using the RegexOptions.IgnoreCase option and similar.

Reply:
I'll do some more research too, and see what I come across for using regex in unity.